src/api/routes/auth/verify/resend.ts

```diff
@@ -24,7 +24,14 @@ const router = Router();
 
 router.post(
 	"/",
-	route({ right: "RESEND_VERIFICATION_EMAIL" }),
+	route({
+		right: "RESEND_VERIFICATION_EMAIL",
+		responses: {
+			204: {},
+			400: {},
+			500: {},
+		},
+	}),
```

Comment:
400 should be APIErrorResponse as should 500, I think? going off the error handler middleware impl